views/syncgroup-page.twig

```diff
@@ -130,6 +130,8 @@
             { "data": "owner", responsivePriority: 3 },
             { "data": "modifiedByName", responsivePriority: 4 },
             { "data": "syncPublisherPort", responsivePriority: 3 },
+            { "data": "syncSwitchDelay", responsivePriority: 3 },
```

Comment:
You will also need to add some table headings for these fields in the HTML output above

Reply:
Thanks again for the review! Table headings has been added on this [commit](https://github.com/xibosignage/xibo-cms/pull/2564/commits/d2bf44bfdc0bae292e60291730cf72b9b009b411).